astropy/utils/tests/test_decorators.py

```diff
@@ -289,6 +289,7 @@ def test1(overwrite):
             assert method(clobber=1) == 1
             assert len(w) == 1
             assert '1.3' in str(w[0].message)
+            assert 'utils/tests/test_decorators.py' in str(w[0].filename)
```

Comment:
Even simpler, I removed the beginning of the path, the goal is just to know the warning does not come from `astropy.utils.decorators`.